tests/test_snql_api.py

```diff
@@ -134,10 +136,28 @@ def test_sub_query(self) -> None:
             ),
         )
         data = json.loads(response.data)
-
         assert response.status_code == 200, data
         assert data["data"] == [{"avg_count": 1.0}]
 
+    def test_max_limit(self) -> None:
```

Comment:
Should this go in `test_discover_api.py`? It looks like there's a similar test in `test_api.py` for events

Reply:
Since this causes the query to fail without ever generating any SQL, the `compare` parameter will always fail. So I put the test here to test the SnQL version and moved the original into the set of legacy tests.